scripts/simapp/env

```diff
@@ -0,0 +1,4 @@
+# Choose from https://hub.docker.com/r/cosmwasm/simapp/tags
+REPOSITORY="cosmwasm/simapp"
+VERSION="latest"
+CONTAINER_NAME="simapp"
```

Comment:
Can we use the one from the Cosmos SD team? https://github.com/cosmos/cosmos-sdk/pull/6517#issuecomment-652901250 There we also get pinned versioning

Reply:
`interchainio/cosmos-sdk` is simapp?

Reply:
It should be, yes